src/views/hooks/useDefaultSender.jsx

```diff
@@ -8,7 +8,7 @@ export function useDefaultSender() {
 
   useEffect(() => {
     const sub = contactsManager?.defaultSender$
-      .pipe(filter((i) => !!i))
+      //.pipe(filter((i) => !!i)) This line never let the value through.
```

Comment:
we must discuss the theory and dataflow behind rxjs and why this kind of pipe operation is helpful

this pipe filters the observable so that our subscription only observes "senders" that are considered valid within the context of the kind of operation we want to perform